caret_demos/src/end_to_end_sample.cpp

```diff
@@ -139,6 +139,14 @@ class SensorDummy : public rclcpp::Node
   SensorDummy(std::string node_name, std::string topic_name, int period_ms)
   : Node(node_name)
   {
+    this->declare_parameter<bool>("use_rosbag", false);
+    bool use_rosbag;
```

Comment:
here, a warning occurs.

```
/home/hasegawa/ros2_ws/src/CARET_demos/caret_demos/src/end_to_end_sample.cpp:146:5: warning: ‘use_rosbag’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  146 |     if (use_rosbag) {
```

Reply:
Thank you for pointing it out. The warning doesn't appear in my environment but the new commit should fix it.

Also, thank you for your idea using clock_publisher. It looks nice to have it but I think most users will perform rosbag record/play when using use_sim. So, I'd like to keep this procedure as of current implementation.